src/ynEIGEN/TokenStakingNode.sol

```diff
@@ -28,12 +30,6 @@ interface ITokenStakingNodeEvents {
     event DeallocatedTokens(uint256 amount, IERC20 token);
 }
 
-interface IYieldNestStrategyManager {
```

Comment:
no need as far as i can tell to have a separate interface

the separation causes testime/runtime failures in case function signatures change